actions/get-changed-files/index.js

```diff
@@ -18,8 +18,12 @@ const getBaseAndHead = (context, core) => {
     }
 };
 
-module.exports = async ({github, context, core}) => {
-    const absolute = !!core.getInput("absolute", {required: false});
```

Comment:
I assume this went away to make the filtering based on directory easier.  I'm curious why we were using absolute paths before.

Reply:
So we weren't using the `absolute` option anywhere (I had added to interact with a previous version of some actions which we're not using anymore)